src/Config/Config.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * Slim Framework (https://slimframework.com)
+ *
+ * @license https://github.com/slimphp/Slim-Console/blob/0.x/LICENSE.md (MIT License)
+ */
+
+declare(strict_types=1);
+
+namespace Slim\Console\Config;
+
+use InvalidArgumentException;
+
+class Config
+{
+    public const SLIM_CONSOLE_BOOTSTRAP_DIR = 'SLIM_CONSOLE_BOOTSTRAP_DIR';
+    public const SLIM_CONSOLE_INDEX_DIR = 'SLIM_CONSOLE_INDEX_DIR';
+    public const SLIM_CONSOLE_INDEX_FILE = 'SLIM_CONSOLE_INDEX_FILE';
+    public const SLIM_CONSOLE_ROOT_DIR = 'SLIM_CONSOLE_ROOT_DIR';
```

Comment:
@adriansuter
I think this can be removed as an env variable. In which instance would you manually declare this? right?

Reply:
I think so.